stablehlo/dialect/TypeInference.cpp

```diff
@@ -3714,6 +3714,23 @@ LogicalResult verifyDynamicBroadcastInDimOp(
                                " does not refer to a "
                                "valid operand dimension");
 
+  if (SmallVector<int64_t> shape;
+      operandType.hasStaticShape() &&
+      matchInts(outputDimensions, shape).succeeded()) {
+    for (int64_t i = 0; i != bcastDimensionsSize; ++i) {
```

Comment:
I recently found out, but there's a `llvm::enumerate` to replace this.

```
for (auto [i, dimIndex] : llvm::enumerate(broadcastDimensions)
...
```

Reply:
Good catch, thanks!